### DISCUSSION TITLE
Adding Linux, windows and mac template

### DISCUSSION THREAD

**Discussion by youssefKadaouiAbbassi** [nylo-core/nylo#25] [Ideas]
Hey,

Is there a reason why the current template does not include Linux, windows and mac? I tried adding them myself, but it seems to be a problem when compiling. Please let me know if you are planning on adding them.

**Comment by agordn52** [comment]
I originally started the project only for IOS/Android but since then I know "Linux, windows and mac" are support now. 
It would be a nice addition to the micro-framework if it's easy to implement.

What errors do you see on compile?

**Reply by quocdat1804** [reply]
Hi
I'm using nylo for flutter mac app. NyStorage doesn't work on mac

**Reply by quocdat1804** [reply]
I found out Nylo uses the flutter_secure_storage package. I configured it with their instructions and now NyStorage worked perfectly
Thank you
